management/server/rule.go

```diff
@@ -0,0 +1,97 @@
+package server
+
+import (
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// TrafficFlowType defines allowed direction of the traffic in the rule
+type TrafficFlowType int
+
+const (
+	// TrafficFlowBidirect allows traffic to both direction
+	TrafficFlowBidirect TrafficFlowType = iota
+)
+
+// Rule of ACL for groups
+type Rule struct {
+	// ID of the rule
+	ID string
+
+	// Name of the rule visible in the UI
+	Name string
+
+	// Source list of groups IDs of peers
+	Source []string
```

Comment:
Do you think it would make sense to have individual peers as well?
I also see it as having groups and peers

Reply:
It is logical if this use case is not rare, because it adds some mess to UI and implementation.

Reply:
so we skip this for now as agreed